src/pages/LoginSignupPage/LoginPage.tsx

```diff
@@ -0,0 +1,15 @@
+import Login from '@components/LoginSignupPage/Login';
+
+const LoginPage = (): JSX.Element => {
+  const handleLogin = (userId: string) => {
+    console.log(`User logged in: ${userId}`);
+  };
+
+  return (
+    <div>
+      <Login onLogin={handleLogin} />
+    </div>
```

Comment:
```suggestion
    <Login onLogin={handleLogin} />
```

따로 의미가 없다면 태그 중첩 안 해주고, 컴포넌트로만 구성해도 될 것 같아보이네용?